src/translations/screens/Assistant.ts

```diff
@@ -75,13 +76,18 @@ const AssistantTexts = {
           ),
         walkLabel: (time: string) => _(`Gå ${time}`, `Walk ${time}`),
       },
+      destination: {
+        a11yLabel: _('Destinasjon'),
+      },
       waitRow: {
         label: _('Vent'),
       },
+      footer: {
+        fromLabel: (place: string, time: string) => _(`Fra ${place} ${time}`),
+        detailsLabel: _('Detaljer'),
+        detailsHint: _('Aktivér for å  flere detaljer om reisen'),
+      },
       journeySummary: {
-        situationsWarning: _('Driftsmeldinger gjelder for dette forslaget.'),
```

Comment:
Is it now no longer indications that there are some situations relevant for the result item?

Reply:
Ah. Probably catched by `hasSituationsTip`